proxyvisor/proxyvisor.go

```diff
@@ -0,0 +1,19 @@
+package proxyvisor
+
+func Provision() (string, error) {
+	return "PIo0c6fYkQN0liVOPAVs7WA4WtP8NYChlDjRKPO75xpmT1Kj0wk1nPJFIwDStv", nil
```

Comment:
What is this hardcoded value? If this is a test, best to put in a `proxyvisor_test.go` file. Is this value in any way private?